h/static/scripts/annotator/sidebar.coffee

```diff
@@ -133,6 +133,9 @@ module.exports = class Sidebar extends Host
         this.hide()
 
   show: ->
+    if this.plugins.CrossFrame
```

Comment:
A `Sidebar` always has a `CrossFrame` plugin, and it's referenced as `@crossframe`, so:

`@crossframe.call('sidebarOpened')`